src/components/reviews-list/ReviewsList.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Avatar from '@mui/material/Avatar'
+import Divider from '@mui/material/Divider'
+import ReviewCard from '~/components/review-card/ReviewCard'
+import { styles } from '~/components/reviews-list/ReviewsList.styles.js'
+import { useTranslation } from 'react-i18next'
+
+const ReviewsList = ({ offer }) => {
+  const { t } = useTranslation()
+  const [visibleCount, setVisibleCount] = useState(1)
+
+  const showSubjectLevel =
```

Comment:
It is usually better to use i18n rather than string concatenation.
As an example, you can look at this [usage](https://github.com/iva-stasia/s2s-frontEnd/blob/8dc8eb8e817dc181502bc89dc90a786b0be923e4/src/components/file-uploader/FileUploader.jsx#L64) and [translation](https://github.com/iva-stasia/s2s-frontEnd/blob/8dc8eb8e817dc181502bc89dc90a786b0be923e4/src/constants/translations/en/constant.json#L2).